### DISCUSSION TITLE
Frigate+ availability

### DISCUSSION THREAD

**Discussion by monsieurlatte** [blakeblackshear/frigate#6612] [Ask A Question]
Someone told me that frigate plus would be better for me with some missed detections of a grey cat in night mode cameras.  Wish I had seen this sooner!   Is the submitting to your own models thing working yet?  I am using a Coral USB device.  Would subscribing now help the devs now or wait till it comes out?

**Comment by NickM-27** [comment]
Subscriptions won't be available until the models are ready. 

A base model is made from the communities submissions and then it is tuned from your submissions.

**Comment by monsieurlatte** [comment]
Got it, so at this point we'd be just sending submissions to frigate + now for the upcoming models?   Any submission we do now doesn't actually help what we are currently using, but subscribing and submitting helps the future models get better and to support the product growth?

**Reply by NickM-27** [reply]
Currently you upload and annotate images. Frigate+ models will be available in a closed beta soon and I believe it has been said the full release will happen alongside 0.13. 

The current model is from Google as a demo model trained on COCO so it is entirely separate to what Frigate+ is.

**Reply by ljmerza** [reply]
are the frigate+ models still happening since the 0.13 PR was opened?

**Reply by NickM-27** [reply]
The PR is in draft, it is simply there to track the release and will be merged when the release is out

**Comment by RexWheeler** [comment]
When annotating images in frigate+ is there a way to add new annotations? Specifically I see Amazon, FedEx and UPS, but I don't see USPS. I would eventually like to be able to know *which* delivery service has come by. I can start uploading and tagging the USPS vehicles that drive by, but it's not in the dropdown list and I don't see a way to add new tags/annotations.

**Reply by NickM-27** [reply]
the user can not add labels, more labels will be added in the future

**Comment by tedder** [comment]
Feature request of sorts: add "false positive" annotations. It'd be nice to be able to help train the "no, that shadow/chair/whatever is not a person" detectors: https://github.com/blakeblackshear/frigate/issues/5883#issuecomment-1492768055

**Reply by tedder** [reply]
aha, it's already coming: https://github.com/blakeblackshear/frigate/pull/6217
